### ISSUE TITLE
InlineDocCommentDeclaration doesn't work with list(...) syntax

### ISSUE THREAD

**Issue by yguedidi** [slevomat/coding-standard#680]
When only one variable in documented from a `list`, it looks like `InlineDocCommentDeclaration` doesn't catch it.

Here an example of a non-compliant code that is not catched:

```php
/** @var $foo Foo */
list($foo, $bar, $baz) = $this->aCall();
```

**Comment by kukulich** [comment]
@yguedidi Could you please try `dev-master` ?

**Comment by yguedidi** [comment]
@kukulich it works!
For info I was on 5.0.4. Any release planned?
Thanks

**Comment by yguedidi** [comment]
@kukulich it looks like that only one `@var` is supported, following syntax breaks:

```php
/** @var $foo Foo */
/** @var $bar Bar */
list($foo, $bar, $baz) = $this->aCall();
```

Error is `Missing variable $foo before or after the documentation comment.`

**Comment by kukulich** [comment]
@yguedidi Try to merge both comments to one.

```
/**
@var Foo $foo
@var Bar $bar
*/
```

**Comment by yguedidi** [comment]
How I didn't thought about that... Thanks!
